api/api.py

```diff
@@ -130,8 +129,8 @@ async def get_notes(project_id):
     escaped_data = [
         [
             note[0],
-            escape(note[1], quote=False),
-            escape(note[2], quote=False),
+            sanitize(note[1]),
+            sanitize(note[2]),
             note[3],
         ]
         for note in data
```

Comment:
no new fixed length array. Please sanitze the whole note dynamically, so that if it ever becomes one entry longer by changing the SQL query this is automatically handled